optuna/samplers/_cmaes.py

```diff
@@ -118,10 +140,11 @@ def __init__(
         independent_sampler: Optional[BaseSampler] = None,
         warn_independent_sampling: bool = True,
         seed: Optional[int] = None,
+        restart_strategy: Optional[str] = None,
+        inc_popsize: int = 2,
         *,
         consider_pruned_trials: bool = False
```

Comment:
Made keyword only arguments.

```suggestion
        *,
        consider_pruned_trials: bool = False,
        restart_strategy: Optional[str] = None,
        inc_popsize: int = 2
```